swift/ios-shoppe-demo/Model/Product.swift

```diff
@@ -21,8 +21,12 @@ class Product {
     init(_ dict: [String: Any]) {
         self.title = dict["title"] as? String ?? ""
         self.description = dict["description"] as? String ?? ""
-        self.price = dict["price"] as? Double ?? -1.0
+        self.price = Decimal(floatLiteral: dict["price"] as? Double ?? -1.0)
```

Comment:
I noticed else where the Doubles are now Decimals.  But this line still looks like it casts `as?` a `Double`.  Is that intentional?

Reply:
no I will fix that thanks so much for catching it!

Reply:
Oh actually I ran into an issue and it was not recognized as a decimal so I needed to cast as a Double the initialize as a Decimal.